webpack.dev.js

```diff
@@ -12,10 +13,13 @@ module.exports = merge(common, {
   plugins: [
     new WebpackShellPlugin({
        onBuildEnd: {
-         scripts: ['node server-build/server.js'],
+         scripts: ['node server-build/server_fbg.js','node server-build/server_bgio.js'],
```

Comment:
Does both execute in parallel in this case?

Reply:
Yes :)